test/Tests/src/Data/Vector_Spec.enso

```diff
@@ -557,11 +557,8 @@ type_spec name alter = Test.group name <|
         alter [Time_Of_Day.new 12, Time_Of_Day.new 10 30] . sort . should_equal [Time_Of_Day.new 10 30, Time_Of_Day.new 12]
         alter [Date_Time.new 2000 12 30 12 30, Date_Time.new 2000 12 30 12 00] . sort . should_equal [Date_Time.new 2000 12 30 12 00, Date_Time.new 2000 12 30 12 30]
 
-        alter ["aa", 2] . sort . should_fail_with Incomparable_Values
-        alter [2, Date.new 1999] . sort . should_fail_with Incomparable_Values
-        alter [Date_Time.new 1999 1 1 12 30, Date.new 1999] . sort . should_fail_with Incomparable_Values
-        alter [Date_Time.new 1999 1 1 12 30, Time_Of_Day.new 12 30] . sort . should_fail_with Incomparable_Values
-        Test.expect_panic_with ([3,2,1].to_array.sort_builtin 42) Type_Error
+        ["aa", 2].sort . should_equal [2, "aa"]
```

Comment:
Please use the `alter` prefix to make sure the test runs with `Vector` as well as with `Array` - e.g.:
```
alter ["aa", 2] . sort . should_equal [2, "aa"]
```
please use this `alter` function everywhere in this `_Spec`.

Reply:
OK, done